src/components/AccountDropdown/AccountChildOption.tsx

```diff
@@ -12,9 +12,17 @@ export const AccountChildOption = forwardRef<AccountChildRowProps, 'button'>(
   ({ title, cryptoBalance, symbol, children, ...props }, ref) => {
     const Component = (
       <MenuItemOption ref={ref} {...props}>
-        <Stack direction='row' justifyContent='space-between'>
-          <RawText>{title}</RawText>
-          <Amount.Crypto value={cryptoBalance} symbol={symbol} />
+        <Stack direction='row' justifyContent='space-between' fontSize='sm' spacing={4}>
+          <RawText fontWeight='bold' whiteSpace='nowrap'>
+            {title}
```

Comment:
All caps is pretty rough here. Can we use title case, e.g. `Segwit Native`?

Reply:
Agreed that could get done as a low lift and would look much better e.g:

```typescript
'SEGWIT NATIVE'.split(' ').map(x => x.toLowerCase()).map(_.upperFirst).join(' ')
// => 'Segwit Native'

'SEGWIT'.split(' ').map(x => x.toLowerCase()).map(_.upperFirst).join(' ')
// => 'Segwit'
```

(can obviously be refactored with a nice LTR pipeline)

Reply:
fixed at the root cause https://github.com/shapeshift/web/pull/2625/commits/6eaaca2d136c1f6ecb44a1680e6698fefeeba5d8